ivy/func_wrapper.py

```diff
@@ -1260,36 +1260,6 @@ def _dtype_device_wrapper_creator(attrib, t):
     """
 
     def _wrapper_outer(version_dict, version, exclusive=True):
-        typesets = {
-            "valid": ivy.valid_dtypes,
-            "numeric": ivy.valid_numeric_dtypes,
-            "float": ivy.valid_float_dtypes,
-            "integer": ivy.valid_int_dtypes,
-            "unsigned": ivy.valid_uint_dtypes,
-            "complex": ivy.valid_complex_dtypes,
-        }
-
-        for key, value in version_dict.items():
```

Comment:
Try doing the exact operation back in the data_types and it should fix the bug.

Reply:
done